custom_components/frigate/media_source.py

```diff
@@ -1304,7 +1304,11 @@ def _generate_recording_title(
 
             if folder is None:
                 return "Recordings"
-            return dt.datetime.strptime(f"{folder['name']}", "%Y-%m").strftime("%B %Y")
+
+            try:
+                return dt.datetime.strptime(f"{folder['name']}", "%Y-%m-%d").strftime("%B %d, %Y")
```

Comment:
Please add a comment explaining what's going on here, akin to the one added in the prior PR (e.g. referring to the change of directory structure, so we can remember to eventually remove the below old support).

Reply:
yeah, there's a lot of weird issue, I have more changes coming to get this working. Turns out I forgot since we want to display the time in local time we have to convert back to UTC before trying to access the files on the frigate system